.travis.yml

```diff
@@ -136,7 +136,10 @@ jobs:
       os: linux
       dist: xenial
       language: python
-      python: 2.7.14
+      python:
+        - "2.7.14"
+        - "3.6"
```

Comment:
Skip 3.5 since it may require additional coniguration
https://stackoverflow.com/a/41262275